packages/node_modules/@webex/redux-module-media/src/actions.js

```diff
@@ -207,13 +207,15 @@ const getMeeting = ({sdkAdapter: {meetingsAdapter}, destination}) => new Promise
   .then((meeting) => meetingsAdapter.getLocalMedia(meeting.id).toPromise()
     .then(({localAudio, localVideo}) => meetingsAdapter.fetchMeetingTitle(destination)
       .then((title) => {
+        const m = meetingsAdapter.meetings[meeting.id];
+
         // eslint-disable-next-line no-param-reassign
         meetingsAdapter.meetings[meeting.id] = {
           ID: meeting.id,
+          ...m,
           title,
           localAudio,
           localVideo,
-          localShare: null,
           remoteAudio: null,
```

Comment:
Since you are now taking the meeting object from the adapter, remove this other values and just overwrite `title`, `localAudio` and `localVideo`, as those are the only things that need to be overwritten.